e2e_tests/test_browse.py

```diff
@@ -0,0 +1,69 @@
+import os
+
+from playwright.sync_api import Page
+
+username = os.getenv("AYR_TEST_USERNAME")
+password = os.getenv("AYR_TEST_PASSWORD")
+
+
+def test_has_title(page: Page):
+    page.goto("/browse")
+    page.get_by_label("Email address").click()
+    page.get_by_label("Email address").fill(username)
+    page.get_by_label("Password").click()
+    page.get_by_label("Password").fill(password)
+    page.get_by_role("button", name="Sign in").click()
+    assert page.title() == "Browse – AYR - Access Your Records – GOV.UK"
+
+
+def test_sort_dropdown(page: Page):
+    page.goto("/browse")
+    page.get_by_label("Email address").click()
+    page.get_by_label("Email address").fill(username)
+    page.get_by_label("Password").click()
+    page.get_by_label("Password").fill(password)
+    page.get_by_role("button", name="Sign in").click()
+    page.get_by_label("", exact=True).click()
+    page.get_by_role("button", name="Search").click()
+    page.select_option("#sort", "body-a")
```

Comment:
do we want to make any assertions on this test? Otherwise were just showing that these can be selected but nothing about the effect of selecting

Reply:
Once we have sort functionality, this test will be changed also.

Reply:
worth noting in sort ticket

Reply:
Added: https://national-archives.atlassian.net/browse/AYR-564